main.py

```diff
@@ -71,7 +71,16 @@ class HomePageRedirectHandler(base.BaseHandler):
     """
     def get(self):
         if self.user_id and user_services.has_fully_registered(self.user_id):
-            self.redirect(feconf.DASHBOARD_URL)
+            # Get all the explorations associated with the user's account
```

Comment:
Yes, please. In general, code should be accompanied by tests, and in any case you won't be able to submit the PR until all the tests have passed.